src/Altinn.App.Api/Controllers/StatelessPagesController.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.NetworkInformation;
+using System.Threading.Tasks;
+
+using Altinn.App.Common.Models;
+using Altinn.App.Common.Serialization;
+using Altinn.App.PlatformServices.Models;
+using Altinn.App.Services.Implementation;
+using Altinn.App.Services.Interface;
+using Altinn.Platform.Storage.Interface.Models;
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Altinn.App.Api.Controllers
+{
+    /// <summary>
+    /// Handles page related operations
+    /// </summary>
+    [Authorize]
```

Comment:
Do we need to be authorized? Ref. anonymous usage of stateless.

Reply:
Fair point. We can make the endpoint public and present a warning/info in the docs that for stateless apps the endpoint does not have authorization and should not expose/use sensitive data